pages/apis/graphql/cookbooks/clusters.md

```diff
@@ -184,3 +184,21 @@ query getClusterQueueAgent {
 }
 ```
 
+## Associate a pipeline with a cluster
+
+First, [get the Cluster ID](#get-a-pipelines-id) you want to associate the Pipeline with.
+Second, [get the Pipeline's ID](./pipelines#get-a-pipelines-id).
```

Comment:
Probably better to adopt [our consistent method for linking to other pages within the help docs](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#internal-links-to-other-pages).

That way, if we ever need to modify these in future (because content may move around), it makes it much easier to conduct a global search and replace on these links...

```suggestion
Second, [get the Pipeline's ID](/docs/apis/graphql/cookbooks/pipelines#get-a-pipelines-id).
```